docs/src/pages/component-api/Form/FormHelperText.md

```diff
@@ -1,30 +1,26 @@
-# TextFieldLabel
+FormHelperText
```

Comment:
That's the old format. We will have to regenerate the documentation (`yarn docs:build`).